experiments/dsl/src/main/scala/org/apache/gearpump/streaming/dsl/StreamApp.scala

```diff
@@ -100,15 +100,13 @@ class TimeReplayableProducer[T:ClassTag](timeReplayableSource: TimeReplayableSou
 }
 
 class KafkaProducer[T:ClassTag](kafkaConfig: KafkaConfig, converter: Message => T) extends Traverse[T] {
```

Comment:
The changes in this PR to DSL or pipeline modules are just to make them compilable with the new API. I'll open follow-up issues to address them